python/pyarrow/tests/test_array.py

```diff
@@ -666,6 +666,13 @@ def test_struct_from_arrays():
         pa.StructArray.from_arrays([a, b, c], fields=[fa2, fb, fc])
 
 
+def test_struct_array_from_chunked():
+    chunked_arr = pa.chunked_array([[1, 2, 3], [4, 5, 6]])
```

Comment:
Can you add a short comment why this is being tested? Something like `# When passing chunked array instead of array, ensure we raise a proper error instead of crashing`

Reply:
👍  done